src/module-1/cards-list-v1/styles_grid_of_cards.css

```diff
@@ -0,0 +1,120 @@
+.grid_container {
```

Comment:
![image](https://user-images.githubusercontent.com/76209817/133897959-fc2e02b2-87a4-4c66-b919-225e4aee1b64.png)
Please check how this page is displayed in Edge browser and fix your styles

Reply:
Sorry, I am a Linux user. I tested in Chrome and Firefox. I do not have the opportunity to test in Microsoft Edge.

Reply:
ok, let it be